gravitee-am-service/src/main/java/io/gravitee/am/service/model/NewIdentityProvider.java

```diff
@@ -32,6 +32,9 @@
     @NotNull
     private String name;
 
+    @NotNull
+    private boolean system;
```

Comment:
As this object is used to create an IdentityProvider from the REST resource, we should reject a creation if the system boolean is true in this method https://github.com/gravitee-io/gravitee-access-management/blob/2c9268c79b/gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/environments/domains/IdentityProvidersResource.java#L106

Reply:
Agree to remove boolean system from the NewIdentityProvider object.